debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/QueueSegment.java

```diff
@@ -109,8 +124,10 @@ public void close() throws IOException {
         writer.close();
     }
 
-    public void sync() throws SyncFailedException {
+    public void sync() throws IOException{
         fd.sync();
+        // TODO: is Files.size going to be slow? Maybe just use byteCount?
```

Comment:
@amit-yb Not 100% sure of this. 
1. I assume this is just reading the inode to get the size. is that going to be a perf concern?
2. doc says `The size may differ from the actual size on the file system due to compression, support for sparse files, or other reasons` .

Reply:
Don't worry about the perf implications of `Files.size()`. It will read from an in-memory inode.